mono/metadata/assembly.c

```diff
@@ -3254,6 +3254,18 @@ mono_assembly_get_image (MonoAssembly *assembly)
 	return assembly->image;
 }
 
+/**
```

Comment:
> This is missing documentation on memory management.

I didn't include it as the only precedent for mentioning memory management in embedding API documentation seems to be "don't". We can certainly change improve precedent, though.

> Could the returned pointer be const?

Probably.